vnext/Desktop.ABITests/NapiTests.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

The file name should be JSI tests - it does not run NAPI tests - it runs JSI tests for the JSI implemented on top of the NAPI.